frontend/server/src/Controllers/Course.php

```diff
@@ -820,7 +823,7 @@ private static function addProblemToAssignment(
             throw new \OmegaUp\Exceptions\NotFoundException('problemNotFound');
         }
 
-        [$masterCommit, $currentVersion] = \OmegaUp\Controllers\Problem::resolveCommit(
+        list($masterCommit, $currentVersion) = \OmegaUp\Controllers\Problem::resolveCommit(
```

Comment:
```suggestion
        [$masterCommit, $currentVersion] = \OmegaUp\Controllers\Problem::resolveCommit(
```

esto no funciona? si no, por qué?

Reply:
Ah, si funciona.